rust-runtime/aws-smithy-compression/src/http.rs

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//! Checksum support for HTTP requests and responses.
+
+/// Support for the `http-body-0-4` and `http-0-2` crates.
+#[cfg(feature = "http-body-0-4-x")]
+pub mod http_body_0_4_x {
+    use crate::Compression;
+    use http_0_2::header::{HeaderMap, HeaderName, HeaderValue};
+
+    /// Implementors of this trait can be used to compress HTTP requests.
+    pub trait RequestCompressor: Compression + Send + Sync {
```

Comment:
Isn't `Compression` already `Send + Sync`?

Reply:
True, I suppose I don't need to include these.